components/login/index.jsx

```diff
@@ -48,6 +52,13 @@ export default function Login() {
   const INFO_TOOLTIP_TEXT = "This is where you signed up for a Solid Pod";
   const INFO_BUTTON_LABEL = "Where is your Pod hosted?";
 
+  const oidcSupported = useClientId(PROVIDER_IRI);
+
+  const authOptions = {
+    clientName: "Inrupt PodBrowser",
```

Comment:
This should be a constant exported at the top of the file (most magic strings should be.)

Reply:
If `Inrupt PodBrowser` is used multiple times, it belongs in a constants file so it can be re-used.

Reply:
As per this comment, I moved `CLIENT_NAME` as well as `PUBLIC_OIDC_CLIENT` into a constants file as they are being reused.

Reply:
As per this comment, I moved `CLIENT_NAME` as well as `PUBLIC_OIDC_CLIENT` into a constants file as they are being reused.